includes/class-piklist-add-on.php

```diff
@@ -230,7 +233,7 @@ public static function current()
 
         if (isset(piklist::$add_ons[$add_on]))
         {
-          return str_replace(DIRECTORY_SEPARATOR, '/', $add_on);
+		  return $add_on;
```

Comment:
Please fix the spacing. You've used tabs throughout the code and this codebase uses spaces. Please convert all the tabs to spaces and make sure the spacing looks correct.